src/index.ts

```diff
@@ -33,17 +33,23 @@ export function getServiceData(apiName: string, beaconName: string, chain: strin
   const goBeaconData = goSync(() =>
     require(`@api3dao/operations/data/apis/${apiName}/beacons/${sanitiseFilename(beaconName)}.json`)
   );
-  if (!isGoSuccess(goBeaconData)) throw new Error(`Service data does not exist.`);
+  if (!isGoSuccess(goBeaconData)) throw new Error(`Beacon service data does not exist.`);
   const beaconData = goBeaconData.data;
 
+  const goTemplateData = goSync(() =>
```

Comment:
I just noticed that this was referring the beacon parameters and not the template parameters. It should have been renamed better initially, but I guess since we won't be using services atm, we can let it be and come back and refactor it later

Reply:
Hey, yeah, I didn't create this repo, but it's good that you noticed this :)

AFAIK we won't use services again (ops will be the new reference for beacons v0.2), so this is just a caretaker action essentially, as you noticed. Thanks for reviewing :)